src/components/checkbox/Checkbox.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Tick from '../../icons/GlyphSmallTick'
+import './Checkbox.css'
+
+class Checkbox extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    onChange: PropTypes.func.isRequired
+  }
+
+  state = {
+    checked: false
+  }
+
+  onClick = () => {
+    this.props.onChange(!this.state.checked)
+    this.setState({ checked: !this.state.checked })
+  }
+
+  render () {
+    let { className, onChange,  ...props } = this.props
+    let content = null
+    className = `Checkbox pointer br1 w1 h1 ${className}`
+
+    if (this.state.checked) {
+      content = <Tick fill='#69C4CD' viewBox='25 25 50 50' />
+      className += ' checked'
+    }
+
+    return (
+      <div

```

Comment:
In general, this is nice code. One improvement would be to implement it as a hidden checkbox input wrapped in a label, then style the label as you have this div. That way you get all the behaviour of a real input element for free.